### PR TITLE
Jessi Nygren Walhed - Week 11 - Project-happy-thoughts 

### PR DESCRIPTION
**Pull request by hemmahosjessi**
Link to my deployed project:
https://babydontworry.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+
+const Header = ({ heading, backgroundColor}) => {
+
+
+    return (
+        <header style={{backgroundColor: backgroundColor}} className="header">
+            <div className='heart-animation'></div>
+            <h1>{heading}</h1>
+            <p className='preamble'>Happiness is an emotional state characterized by feelings of joy, satisfaction, contentment, and fulfillment.</p>
```
Comment (katiewu1) [inline code/src/components/Header.js:11]: Like your qoute! ❤️

File: code/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+
+const Header = ({ heading, backgroundColor}) => {
+
+
+    return (
+        <header style={{backgroundColor: backgroundColor}} className="header">
```
Comment (katiewu1) [inline code/src/components/Header.js:8]: Nice that our lab practises come in handy here 💯

File: code/src/components/LikeButton.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// import React from 'react'
```
Comment (katiewu1) [inline code/src/components/LikeButton.js:1]: Maybe can remove the LikeButton.js? Seems like you don't need this file.

File: code/src/components/Loader.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+.loader-overlay h1 {
+    color: #fff;
+}
+
+.loading-heart {
+    position: relative;
+    width: 100px;
+    height: 90px;
+    animation: loaderheartbeat 2s infinite;
+    margin:0 auto
```
Comment (katiewu1) [inline code/src/components/Loader.css:24]: forgot a semicolon after `margin: 0 auto`

File: code/src/components/Loader.css [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    content: "";
+    left: 50px;
+    top: 0;
+    width: 50px;
+    height: 80px;
+    background: #fff;
+    -moz-border-radius: 50px 50px 0 0;
+    border-radius: 50px 50px 0 0;
+    -webkit-transform: rotate(-45deg);
+       -moz-transform: rotate(-45deg);
```
Comment (katiewu1) [inline code/src/components/Loader.css:38]: I think most of the modern browsers support regular selectors, so we don't have to specify -webkit-, -moz-, -ms- and -o- anymore.  For example for the transform selector, we just need to write `transform: rotate(-45deg); `

File: code/src/components/Loader.css [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    -webkit-transform-origin: 100% 100%;
+       -moz-transform-origin: 100% 100%;
+        -ms-transform-origin: 100% 100%;
+         -o-transform-origin: 100% 100%;
+            transform-origin :100% 100%;
+  }
+  
+  
+  
+  @keyframes loaderheartbeat
```
Comment (katiewu1) [inline code/src/components/Loader.css:64]: Love animation!

File: code/src/components/Loader.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import './Loader.css'
+
+const Loader = () => {
+
+    return (
+		<div className="loader-overlay">
+            <h1>HAPPY LOADING . . .</h1>
+			<div className="loading-heart" />
```
Comment (katiewu1) [inline code/src/components/Loader.js:9]: The indentions are not quite aligned here

File: code/src/components/ThoughtForm.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+const ThoughtForm = ({ onFormSubmit, newThought, setNewThought, letterCounter }) => {
+
+
+    return (
+        <form className="form" onSubmit={onFormSubmit}>
+            <label htmlFor='newThought'className='input-label'>What is happiness for you?</label>
```
Comment (katiewu1) [inline code/src/components/ThoughtForm.js:8]: Happiness is EVERYTHING! Life.. Family.. Good Food.. yeah it's very important 😄

File: code/src/components/ThoughtItem.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+
+    return (
+        <div className='thought-item'>
+            <p className='thought-message'>{thought.message}</p>
+            <div className='likesAndTime'>
+                <button 
+                    onClick={() => onLikesChange(thought._id)} 
+                    className='btn'>
+                    <span style={{backgroundColor: thought.hearts > 0 ? '#F3B1AF' : '#EAEAEA'}} className='btn-heart' role='img' aria-label='heart emoji'>❤️</span> x {thought.hearts}
```
Comment (katiewu1) [inline code/src/components/ThoughtItem.js:16]: Nice use of ternary operator! 💯

File: code/src/index.css [hunk 10]
```diff
@@ -1,6 +1,14 @@
+html {
+  box-sizing: border-box;
+}
```
Comment (katiewu1) [inline code/src/index.css:3]: Does it work this way too? Usually I either write 
```
* {
  box-sizing: border-box;
}
```
or
```
*, *:before, *:after {
  box-sizing: border-box;
}
```

File: code/src/components/Footer.js [hunk 11]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const Footer = () => {
+
+return (
+    <footer className='footer'>
+        <p className='footer-p'>
+            Don´t worry, about a thing. Cause, every little thing, is gonna be alright!
```
Comment (katiewu1) [inline code/src/components/Footer.js:8]: 🎶 😄 🎧

**Review by katiewu1 — COMMENTED** [review]
Hi Jessi! Nice page! You automatically become really happy when seeing your page 😄  Like the quotes, and you can 'almost' hear the background music when reading the short lyrics🎶   
The heart emoji is not centered in Chrome for me (but it work perfectly in Edge and Safari). I have the same issue with my page, don't know if it's something bugging with Chrome or my Mac... Is it the same for you?
Overall, I don't have any major comments about your page. I only left some comments about small details. Hopefully you find them useful! 
Great job Jessi! 🦊 💪

File: code/src/components/Loader.css (hunk 5 above)
Comment (hemmahosjessi) [reply code/src/components/Loader.css:38]: Oh good to know! Thank you!

File: code/src/components/Loader.js (hunk 7 above)
Comment (hemmahosjessi) [reply code/src/components/Loader.js:9]: Right! didn't put the time to go through it. Will  think about that! Thank you for reminding me 😃

File: code/src/components/ThoughtForm.js (hunk 8 above)
Comment (hemmahosjessi) [reply code/src/components/ThoughtForm.js:8]: ❤️

File: code/src/index.css (hunk 10 above)
Comment (hemmahosjessi) [reply code/src/index.css:3]: It worked... I think 😅 but yes, the * is probably better to use. Not sure about the _before, and _after. Maybe you can tell me sometimes.

File: code/src/components/Header.js (hunk 1 above)
Comment (hemmahosjessi) [reply code/src/components/Header.js:11]: Thank you for the review!

**Pull request closed without merging**